lib/crystalball/rspec/prediction_pruning/examples_pruner.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module Crystalball
+  module RSpec
+    module PredictionPruning
+      # A class to prune given world example groups to fit the limit.
+      class ExamplesPruner
+        attr_reader :world, :to
+
+        # @param [RSpec::Core::World] rspec_world RSpec world instance
+        # @param [Integer] to upper bound limit for prediction.
+        def initialize(rspec_world, to:)
+          @world = rspec_world
+          @to = to
```

Comment:
The keyword argument `to:` is cool, but you can give a more meaningful name to the ivar, like `limit`, `examples_limit` or `upper_bound_limit`.